models/engine/db_storage.py

```diff
@@ -77,13 +77,13 @@ def get(self, cls, id):
             return self.all(cls).get(cls + '.' + id)
 
     def count(self, cls=None):
-        """ Return number objs in strg depend match of class name """
+        """  """
```

Comment:
Add comments for count method and description of variables

Reply:
Ok, thank